addons/website/static/tests/tours/text_animations.js

```diff
@@ -0,0 +1,37 @@
+/** @odoo-module */
+
+import wTourUtils from "website.tour_utils";
+
+wTourUtils.registerWebsitePreviewTour("text_animations", {
+    test: true,
+    url: '/',
+    edition: true,
+}, [
+    wTourUtils.dragNDrop({
+        id: "s_cover",
+        name: "Cover",
+    }),
+    {
+        content: "Click on the snippet title",
+        trigger: 'iframe .s_cover h1 > font',
+        run: "dblclick", // Make sure the title is selected.
```

Comment:
IDK if you wanted to use the new or the old way to use quotes but it doesn't look consistent. I would go with double quote everywhere :)